content/en/docs/cheatsheets/bootstrap.md

```diff
@@ -445,3 +445,40 @@ patches:
       kind: Kustomization
       name: "flux-system"
 ```
+
+### Enable notifications for third party controllers
+
+Enable notifications for 3rd party Flux controllers such as [tf-controller](https://github.com/weaveworks/tf-controller):
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - gotk-components.yaml
+  - gotk-sync.yaml
+patches:
+  - patch: |
+      - op: add
+        path: /spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/eventSources/items/properties/kind/enum/-
+        value: Terraform
+    target:
+      kind: CustomResourceDefinition
+      name:  alerts.notification.toolkit.fluxcd.io
+  - patch: |
+      - op: add
+        path: /spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/resources/items/properties/kind/enum/-
+        value: Terraform
+    target:
+      kind: CustomResourceDefinition
+      name:  receivers.notification.toolkit.fluxcd.io
+  - patch: |
+      - op: add
+        path: /rules/-
+        value:
+          apiGroups: [ 'infra.contrib.fluxcd.io' ]
+          resources: [ '*' ]
+          verbs: [ '*' ]
+    target:
+      kind: ClusterRole
+      name:  crd-controller
```

Comment:
There's no `crd-controller` inside my `gotk-components.yaml`, but there is `crd-controller-flux-system`.
I guess there was a mechanism that added the namespace as a suffix to the ClusterRole, right?

Reply:
Yes good catch, I’ll fix it. Thanks